examples/example-bukkit/src/main/java/org/incendo/cloud/examples/bukkit/builder/feature/minecraft/ComponentExample.java

```diff
@@ -0,0 +1,57 @@
+//
+// MIT License
+//
+// Copyright (c) 2024 Incendo
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+//
+package org.incendo.cloud.examples.bukkit.builder.feature.minecraft;
+
+import net.kyori.adventure.text.minimessage.MiniMessage;
+import org.bukkit.command.CommandSender;
+import org.checkerframework.checker.nullness.qual.NonNull;
+import org.incendo.cloud.bukkit.BukkitCommandManager;
+import org.incendo.cloud.examples.bukkit.ExamplePlugin;
+import org.incendo.cloud.examples.bukkit.builder.BuilderFeature;
+import org.incendo.cloud.minecraft.extras.parser.ComponentParser;
+import org.incendo.cloud.parser.standard.StringParser;
+
+/**
+ * Example showcasing the component parser from cloud-minecraft-extras.
+ */
+public final class ComponentExample implements BuilderFeature {
+
+    @Override
+    public void registerFeature(
+        final @NonNull ExamplePlugin examplePlugin,
+        final @NonNull BukkitCommandManager<CommandSender> manager
+    ) {
+        manager.command(
+            manager.commandBuilder("builder")
+                .literal("minimessage")
+                .required("msg", ComponentParser.componentParser(
+                    MiniMessage.miniMessage()::deserialize,
```

Comment:
```suggestion
                    MiniMessage.miniMessage(),
```
this should work too, right?

Reply:
Or even the miniMessageParser shorthand... I guess this is fine for the example to show using a generic function.

Reply:
yeah. How many examples usages do you think I should add there?